includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1760,6 +1785,41 @@ private function add_order_info_to_object( $charge_id, $object ) {
 		return $object;
 	}
 
+	/**
+	 * Formats dispute for UI
+	 *
+	 * @param  array $dispute The dispute coming from API.
+	 * @return array array in the format that the UI expects
+	 */
+	private function format_dispute_for_ui( array $dispute ): array {
```

Comment:
This method is never used and can be removed.

Reply:
Removed.